src/scanners/css.js

```diff
@@ -0,0 +1,67 @@
+import BaseScanner from 'scanners/base';
+import * as cssParser from 'css';
+import * as rules from 'rules/css';
+import { CSS_SYNTAX_ERROR } from 'messages';
+import { VALIDATION_ERROR } from 'const';
+import { ignorePrivateFunctions } from 'utils';
+
+
+export default class CSSScanner extends BaseScanner {
```

Comment:
Yeah I think we could, I just didn’t get to it right away. I’ll spend some more time today on that and see what I can get.